controllers/seekers.js

```diff
@@ -0,0 +1,148 @@
+const models = require('../models')
+
+const getAllSeekersWithTitles = async (req, res) => {
+  try {
+    let seekers
+    let lodestar = req.params ? req.params.lodestar : undefined
```

Comment:
Could use `&&` here instead. If there is a value you'll get it, otherwise you'll get undefined.

```JavaScript
let loadstar = req.params && req.params.lodestar
```